libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -116,8 +128,8 @@ export class SkyActionButtonContainerComponent
   }
 
   public ngOnDestroy(): void {
-    this.ngUnsubscribe.next();
-    this.ngUnsubscribe.complete();
+    this.#ngUnsubscribe.next();
+    this.#ngUnsubscribe.complete();
   }
 
   public onContentChange() {
```

Comment:
I know this wasn't called out before - but care to put return types on functions that are missing one?

Reply:
added